cli/examples.go

```diff
@@ -46,17 +47,68 @@ func runGenExamplesCmd(cmd *cobra.Command, args []string) error {
 		example = args[0]
 	}
 
-	switch example {
-	case "startrek":
-		fmt.Print(startrekSQL)
-		fmt.Println(footerComment)
-	default:
+	if sql, ok := examples[example]; ok {
+		fmt.Print(sql)
+	} else {
 		return fmt.Errorf("don't know how to generate example data for %q", example)
 	}
-
+	fmt.Println(footerComment)
 	return nil
 }
 
+var examples = map[string]string{
+	"intro":    introSQL,
+	"startrek": startrekSQL,
+}
+
+const introSQL = `
+DROP DATABASE IF EXISTS intro;
+CREATE DATABASE intro;
```

Comment:
Ok for CREATE DATABASE. However I still want to keep DROP TABLE / CREATE TABLE so that we can keep exercising schema updates with these examples.